alws/routers/uploads.py

```diff
@@ -14,15 +22,66 @@
 )
 
 
+async def get_repo_product(
+    session: AsyncSession,
+    repository: str,
+):
+    if repository.endswith("br"):
+        build = (
+            (
+                await session.execute(
+                    select(Build).filter(
+                        Build.repos.any(Repository.name.ilike(f'%{repository}'))
+                    )
+                )
+            )
+            .scalars()
+            .first()
+        )
+        if build:
+            return (
+                (
+                    await session.execute(
+                        select(Product).filter(Product.team_id == build.team_id)
+                    )
+                )
+                .scalars()
+                .first()
+            )
+    else:
```

Comment:
as for me, this else statement is unnecessary if we will return a value in the if block
```python
if repository.endswith("br"):
    ...
    return product
return (await session.execute(...))
```